helpers/formValidation.tsx

```diff
@@ -1,7 +1,6 @@
 import * as Yup from 'yup'
 
 const TEXT_MIN = 2
-const TEXT_MAX = 20
```

Comment:
Why remove `TEXT_MAX` instead of just increase the value? We should have a limit.

Reply:
> Why remove `TEXT_MAX` instead of just increase the value? We should have a limit.

is that an industry practice to define max characters? I have seen limitations on passwords but never with usernames.

Reply:
> > Why remove `TEXT_MAX` instead of just increase the value? We should have a limit.
> 
> is that an industry practice to define max characters? I have seen limitations on passwords but never with usernames.

What are consequences for not having max characters?

Reply:
Maybe it won't do anything.
There is a rule to never trust users input... in general
What will happen if user choose to input a lorem ipsum or whathever? maybe it will work maybe it will break something?
You can keep this control with rules.
But I agree 20 is a bit to low 🙂

Reply:
valid point @Wolfy64 , didn't think of that.

Reply:
I don't think this matters. Since we follow good practice, the password is hashed to a certain length. Even if you hash lorem ipsum, it is still valid.

Reply:
https://github.com/garageScript/c0d3-app/issues/187